examples/base_difference_mapcubes.py

```diff
@@ -0,0 +1,41 @@
+"""
+========================================
+Base Difference in Mapcubes
+========================================
+
+This example illustrates how to do base differencing with a MapCube.
+Base differencing uses a fixed reference point when compared to running difference.
+"""
+
+import matplotlib.pyplot as plt
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
```

Comment:
Can you make sure that all the length of these lines are 79 characters long please.